src/main/java/com/hivemq/cli/DefaultCLIProperties.java

```diff
@@ -0,0 +1,246 @@
+/*
+ * Copyright 2019 HiveMQ and the HiveMQ Community
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+package com.hivemq.cli;
+
+
+import com.hivemq.cli.converters.EnvVarToByteBufferConverter;
+import com.hivemq.cli.converters.FileToByteBufferConverter;
+import com.hivemq.cli.converters.FileToCertificateConverter;
+import com.hivemq.cli.converters.FileToPrivateKeyConverter;
+import com.hivemq.client.mqtt.MqttVersion;
+import org.jetbrains.annotations.NotNull;
+import org.jetbrains.annotations.Nullable;
+import org.pmw.tinylog.Level;
+
+import javax.inject.Singleton;
+import java.io.*;
+import java.nio.ByteBuffer;
+import java.security.PrivateKey;
+import java.security.cert.X509Certificate;
+import java.util.*;
+
+@Singleton
+public class DefaultCLIProperties {
```

Comment:
Why is the class called DefaultCLIProperties when it handles the properties in general

Reply:
ok got it, because you can set properties at runtime and the configured properties are the default

Reply:
Your confusion is justified! 
I will clear things up by Javadocing the class 👍